script.js

```diff
@@ -0,0 +1,37 @@
+let gridCount = 16;
+let grid = document.querySelector("#grid-container");
+const controls = document.querySelector("#controls");
+
+/*
+Non functional reset button :)
+
+const resetBtn = document.createElement("button");
+resetBtn.classList.add("reset-button");
+resetBtn.textContent = "Reset";
+resetBtn.addEventListener("click", () => {
+  cell.className = "cell";
+});
+controls.appendChild(resetBtn); */
+
+function genDivs() {
+  for (
+    let horizontalIndex = 0;
+    horizontalIndex < gridCount;
+    horizontalIndex++
+  ) {
+    let column = document.createElement("div");
+    column.classList.add("column");
+
+    for (let verticalIndex = 1; verticalIndex <= gridCount; verticalIndex++) {
```

Comment:
I recommend sticking to counting from zero because that's the most conventional way in programming. Unless you specifically need index to start from `1` for some other reason.
```suggestion
    let horizontalIndex = 0;
    horizontalIndex < gridCount;
    horizontalIndex++
  ) {
    let column = document.createElement("div");
    column.classList.add("column");

    for (let verticalIndex = 0; verticalIndex < gridCount; verticalIndex++) {
```